src/SortBy/SortBy.jsx

```diff
@@ -1,16 +1,22 @@
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
 
 import styles from './SortBy.scss';
+import { setSortBy } from '../actions';
 
-const SortBy = ({ movieDetails }) => {
+const SortBy = ({ movieDetails, dispatch }) => {
   const cx = classNames.bind(styles);
+  const handleSortBy = (e) => {
+    const sortBy = e.target.dataset.sortBy;
+    dispatch(setSortBy(sortBy));
+  };
   return !(movieDetails) ? (
     <div className={styles.sortByGroup}>
       <span className={styles.sortBy}>Sort by</span>
-      <button className={cx('releaseDay', 'sortByBtn')}>release day</button>
-      <button className={cx('rating', 'sortByBtn', 'active')}>rating</button>
+      <button onClick={handleSortBy} data-sort-by={'releaseDay'} className={cx('releaseDay', 'sortByBtn')}>release day</button>
```

Comment:
it is react :) We don't need to set attributes to elements. You can pass such function to onClick: `onClick={ () => handleSortBy('releaseDay') }` for the first button and  `onClick={ () => handleSortBy('rating') }` for the second